app/src/ui/app.tsx

```diff
@@ -1094,6 +1095,20 @@ export class App extends React.Component<IAppProps, IAppState> {
     this.props.dispatcher.selectRepository(repository)
     this.props.dispatcher.closeFoldout(FoldoutType.Repository)
   }
+
+  private onViewCommitOnGitHub = async (SHA: string) => {
+    const repository = this.getRepository()
+
+    if (!repository || repository instanceof CloningRepository || !repository.gitHubRepository) {
+      return
+    }
+
+    const url = `${repository.gitHubRepository.htmlURL}/commit/${SHA}`
+
+    if (url) {
```

Comment:
Actually, it is, but not really lol. 

I need to check if `repository.gitHubRepository.htmlURL`.